readme.md

```diff
@@ -471,10 +471,10 @@ Alternatively, you can press enter or escape.
 OSARA allows you to read the current audio peak for channels 1 and 2 of either the current or master tracks.
 You do this using the following actions:
 
-- OSARA: Report current peak for channel 1 of current track: j
-- OSARA: Report current peak for channel 2 of current track: k
-- OSARA: Report current peak for channel 1 of master track: shift+j
-- OSARA: Report current peak for channel 2 of master track: shift+k
+- OSARA: Report level in peak dB at play cursor for channel 1 of current track (reports input level instead when track is armed): J
+- OSARA: Report level in peak dB at play cursor for channel 2 of current track (reports input level instead when track is armed): K
+- OSARA: Report peak at play cursor position for channel 1 of master track: Shift+J
```

Comment:
Is there a reason you say "level in peak dB" for normal tracks but just "peak" for master? Why the difference?